config/webpack/environment.js

```diff
@@ -1,3 +1,11 @@
 const { environment } = require('@rails/webpacker')
 
+const webpack = require('webpack')
+environment.plugins.prepend('Provide',
+  new webpack.ProvidePlugin({
+    $: 'jquery',
+    jQuery: 'jquery'
```

Comment:
I recommend not using jQuery. I comment on it in case you want to use it in the future.

Reply:
In this case for flecibility in the team to finish some task more easier and faster we used boostrap, this is the reason to use jquery.